src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/operations/transform.py

```diff
@@ -38,13 +38,15 @@ def add_transform_output(client, account_name, resource_group_name, transform_na
     return client.create_or_update(resource_group_name, account_name, transform_name, transform.outputs)
 
 
-def remove_transform_output(client, account_name, resource_group_name, transform_name, presets):
+def remove_transform_output(client, account_name, resource_group_name, transform_name, output_index):
     transform = client.get(resource_group_name, account_name, transform_name)
 
-    transform_output_list = list(filter(lambda x: not hasattr(x.preset, 'preset_name') or x.preset.preset_name.value not in presets,
-                                        transform.outputs))
+    try:
+        transform.outputs.pop(output_index)
+    except IndexError:
+        raise CLIError("index {} doesn't exist on outputs".format(output_index))
```

Comment:
Missing capital letter at the beggining of the sentence and the period in the end.

Reply:
I wanted the same message as when you do a generic update --remove (without capitalizing first letter)

Reply:
Missing dot at the end, although since it's an error it may be dismissed?

Reply:
I just wanted to keep the same message as this:
![image](https://user-images.githubusercontent.com/8594276/45427007-25b6d200-b674-11e8-92cd-809457964cd1.png)